paywall/src/unlock.js/Wallet.ts

```diff
@@ -92,10 +86,18 @@ export default class Wallet {
     return this.userAccountNetwork
   }
 
+  getHasWallet = () => {
```

Comment:
I intend for these accessors to be temporary. Once you strip the postmessages out of this class, all the real work it does happens in the constructor which means this class can probably be replaced entirely.